content/blog/2024-09-10-unlock-one-million-rps-part2.md

```diff
@@ -0,0 +1,147 @@
++++
+# `title` is how your post will be listed and what will appear at the top of the post
+title= "Unlock 1 Million RPS: Experience Triple the Speed with Valkey - part 2"
+# `date` is when your post will be published.
+# For the most part, you can leave this as the day you _started_ the post.
+# The maintainers will update this value before publishing
+# The time is generally irrelvant in how Valkey published, so '01:01:01' is a good placeholder
+date= 2024-09-11 01:01:01
```

Comment:
We have a great history of launching blogs in the past though.

Reply:
:sigh: This is an annoying issue with this workflow. It's not actually deployed so I'll fix it on a seperate PR.

Reply:
@madolson https://github.com/valkey-io/valkey-io.github.io/pull/133

Reply:
Yeah, I don't know if it really matters if the date is wrong.

Reply:
that's an interesting idea, although it could involve some arcane GitHub Actions hackery.